packages/svelte/tests/server-side-rendering/samples/css-injected-options-nested/Nested.svelte

```diff
@@ -0,0 +1,9 @@
+<svelte:options css="injected" />
+
+<div class="foo">foo</div>
+
+<style>
+	.foo {
```

Comment:
this should use a different class name than the one in `main.svelte` to make the expected outcome more obvious (right now it's two foo classes, hard to visually check)

Reply:
Good idea, will fix immediately

Reply:
Fixed it